highfidelity/hifi
# Fix scriptslog multiply window

https://highfidelity.fogbugz.com/f/filters/177/My-Cases

Test plan:
- Open Developer > Script Log (HMD Friendly) several times.  
- make sure only one window created
- Close and reopen the window. 
- Make sure it still works
